src/Phinx/Migration/AbstractMigration.php

```diff
@@ -66,6 +66,11 @@
     protected $input;
 
     /**
+     * @var bool
+     */
+    protected $goingUp;
```

Comment:
This would not be a boolean or be accurate when using the up and down methods, would it?

Reply:
Fixed.